EpikCord/client/client.py

```diff
@@ -3,7 +3,27 @@
 
 
 class Client:
+    """The main class of EpikCord. Use this to interact with the Discord API and Gateway."""
     def __init__(self, token: str, intents: Intents, *, version: int = 10):
```

Comment:
I dont like magic number, it should be extracted to a final int constant.

Reply:
Yeah I guess that makes sense